client/analytics/report/customers/config.js

```diff
@@ -136,6 +136,81 @@ export const advancedFilters = {
 				} ) ),
 			},
 		},
+		order_count: {
+			labels: {
+				add: 'No. of Orders',
+				remove: 'Remove order  filter',
+				rule: 'Select an order count filter match',
+				title: 'No. of Orders {{rule /}} {{filter /}}',
+			},
+			rules: [
+				{
+					value: 'max',
```

Comment:
The `NumberFilter` component expects one of: `lessthan`, `morethan`, `between`.

Reply:
Thanks for testing @jeffstieler!  I changed these to `min` and `max` to try and match the existing controller params as closely as possible.  Is there something that limits to those values?  

In testing, it seems to work okay with different values for `lessthan` and `morethan` with the exception of `between` that's needed to split those fields into 2.